frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/helpers/diagram-helpers.js

```diff
@@ -84,7 +84,7 @@ export const addPropertyToList = (list, data) => {
   return list
 }
 
-export const createEditLink = (node, overrideKind, overrideCluster, overrideApiVersion) => {
+export const createEditLink = (node, hubClusterName, overrideKind, overrideCluster, overrideApiVersion) => {
```

Comment:
I think `hubClusterName` needs to be passed through to `getEditLink`?

Reply:
Thanks @KevinFCormier ! Yes, good catch. I fixed it now.